src/checkbox.test.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import TestUtils from 'react-addons-test-utils';
+
+import Checkbox from './Checkbox';
+
+
+describe('Checkbox', function() {
+
+	let checkboxComponent,
+		checkbox;
+
+	beforeEach(() => {
+		checkboxComponent = TestUtils.renderIntoDocument(
+			<Checkbox label='Hello!' name='greeting' id='hello' value='hello' />
+		);
+		checkbox = TestUtils.findRenderedDOMComponentWithTag(checkboxComponent, 'input');
+	});
+
+	afterEach(() => {
+		checkboxComponent = null;
+		checkbox = null;
+	});
+
+	it('exists', function() {
+		expect(checkbox).not.toBeNull();
+	});
```

Comment:
because your using findRenderedDOMComponent this wont return null when it fails, instead it throws an error. i would do 
```
expect(() => TestUtils.findRenderedDOMComponentWithType(checkboxComponent, Checkbox)).not.toThrow() 
```
instead